### PR TITLE
LOG-5381: fix bundle to include alert

### PR DESCRIPTION
**Pull request by jcantrill**
### Description
This PR:
* fixes the bundle for the changes introduced by #2438 

### Links
* https://issues.redhat.com//browse/LOG-5381

cc @vparfonov @Clee2691

### CODE REVIEW

File: bundle/manifests/collector_monitoring.coreos.com_v1_prometheusrule.yaml [hunk 1]
```diff
@@ -103,6 +103,18 @@ spec:
       labels:
         service: visualization
         severity: Warning
+    - alert: DiskBufferUsage
+      annotations:
+        message: 'Collectors potentially consuming too much node disk, {{ $value }}% '
+        summary: Detected consuming too much node disk on $labels.hostname host
+      expr: "(label_replace(sum by(hostname) (vector_buffer_byte_size{component_kind='sink',
```
Comment (Clee2691) [inline bundle/manifests/collector_monitoring.coreos.com_v1_prometheusrule.yaml:110]: The expression is not in the format as the other alerts. Does that matter?

Comment (jcantrill) [reply bundle/manifests/collector_monitoring.coreos.com_v1_prometheusrule.yaml:110]: In what way?  I just ran "make bundle" so if the format is off then we need to understand why

**Pull request merged**